views/Authentication/RestorePassword/RestorePassword.tsx

```diff
@@ -68,9 +69,35 @@ const RestorePassword: FC<IRestorePassword> = ({
     else if (clientType === 'admin') return ClientDomainType.AdminAppDomain;
     else return ClientDomainType.CandidateAppDomain;
   };
+  /**
+   * Can add more languages if required
+   */
+  const i18ToLanguage = (language: string) => {
+    switch (language) {
+      case 'de':
+        return Language.German;
+      case 'es':
+        return Language.SpanishCastilian;
+      case 'in':
+        return Language.Indonesian;
+      case 'fr':
+        return Language.French;
+      case 'tr':
+        return Language.Turkish;
+      default:
+        return null;
+    }
+  };
```

Comment:
Have you checked sz-candidate-client/src/constants/languages.ts for this kind of mapping? I can see maps which are handling this if I am not wrong.